<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstFrameDetectMethodsSetDialog</class>
 <widget class="QDialog" name="FirstFrameDetectMethodsSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>First Frame Detect Methods</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QRadioButton{

font-weight: bold;
font-size: 15px;

}

QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QLabel{
border: none;
background: white;
font-weight: bold;
font-size: 15px;

}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_hog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HOG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_lbp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>LBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_haar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>59</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HAAR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>50</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
